packages/eslint-plugin/tests/rules/prefer-literal-enum-member.test.ts

```diff
@@ -73,6 +73,8 @@ enum Foo {
   E = 1 & 0,
   F = 1 ^ 0,
   G = ~1,
+  E = G | 1,
+  F = G | E,
```

Comment:
[Testing] Let's add a new test case for each of these two added cases: https://typescript-eslint.io/contributing/local-development#granular-unit-tests

Here and below.

Reply:
I'm sorry, I don't think I understand what is being asked here

Reply:
Ah sorry - what I mean to say is:

1. Each of the entries in the `invalid` and `valid` is its own "unit test"
2. In theory, we like having each unit test check for a specific behavior
3. The added code is testing a newly added behavior
4. Therefore, the added code should be added in a new unit test (new entries in the `invalid` or `valid` arrays)

The existing code in our repo doesn't always respect the one-behavior-per-unit-test preference... but new code should ideally.

Does that make sense?